src/components/DetailsMetrics/DetailsMetrics.scss

```diff
@@ -14,28 +15,38 @@
     color: $primary;
   }
 
-  &__card-tmp {
-    z-index: 1;
-    top: 0;
-    position: sticky;
-    box-sizing: border-box;
-    border: $primaryBorder;
-    border-radius: 12px;
-    background: $white;
-    padding-top: 12px;
-    width: 100%;
-    transition: 0.1s;
-    animation: ease-in-out;
-    max-height: 140px;
-  }
-
   &__card {
     border-radius: 12px;
     border: $primaryBorder;
     padding: 6px 16px 35px;
-    transition: ease-in-out;
     width: 100%;
 
+    &-invocations {
+      z-index: 1;
+      align-self: baseline;
+      top: 0;
+      position: sticky;
+      box-shadow: 0 -10px 0 0 $white;
+      border-radius: 12px;
+      background: $white;
+      padding-top: 12px;
+      padding-bottom: 0;
+      width: 100%;
+
+      &-header-hidden .stats-card__row {
+        height: 0;
+        min-height: 0 !important;
+        visibility: hidden !important;
+        transition: all 0.2s ease-in-out;
+        margin-bottom: 0 !important;
```

Comment:
don't use `!important` here. We should use it only in rare cases such as library styles overwrites.

Reply:
done